terraform/projects/infra-jump-instance/main.tf

```diff
@@ -0,0 +1,109 @@
+## Variables
+
+variable "additional_tags" {
+  type        = "map"
+  description = "Stack specific tags to apply"
+  default     = {}
+}
+
+variable "aws_region" {
+  type        = "string"
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "remote_state_bucket" {
+  type        = "string"
+  description = "S3 bucket we store our terraform state in"
+  default     = "ecs-monitoring"
+}
+
+variable "stack_name" {
+  type        = "string"
+  description = "Unique name for this collection of resources"
+  default     = "ecs-monitoring"
+}
+
+variable "enable_jumpbox" {
+  type        = "string"
+  description = "Boolean, when true a jumpbox will be deployed"
+  default     = false
```

Comment:
https://www.terraform.io/docs/configuration/variables.html#booleans

Reply:
Don't think this is used